xml/slurm.xml

```diff
@@ -76,16 +76,16 @@
       directory.
      </para>
      <para>
-      Configure, enable and start <emphasis>munge</emphasis> on the control and
+      Configure, enable and start <command>munge</command> on the control and
       compute nodes as described in <xref linkend="sec-remote-munge"/>.
      </para>
     </step>
     <step>
      <para>
       Install <package>slurm-node</package> on the compute nodes with
-      <command>zypper in slurm-node</command> and <package>slurm</package> on
-      the controll node with <command>zypper in slurm</command>. The package
-      <package>munge</package>will be installed as dependence automatically.
+      <command>zypper in slurm-node</command>, and <package>slurm</package> on
+      the control node with <command>zypper in slurm</command>. The package
+      <package>munge</package>will automatically be installed as a dependency.
```

Comment:
Even if the [MUNGE](https://dun.github.io/munge/) project name will probably be fixed in a later revision, it might make sense to fix the space before "will" here already:

```suggestion
      <package>munge</package>will automatically be installed as a dependency.
```

Reply:
The comment here is still relevant, the space before "will" is still missing.

Reply:
Already fixed. I do not know why Github is not showing it, but I had fixed it and pushed it.